packages/google_maps_flutter/google_maps_flutter_android/test/google_maps_flutter_android_test.dart

```diff
@@ -104,7 +104,7 @@ void main() {
     expect(bounds, expectedBounds);
   });
 
-  test('moveCamera calls through', () async {
+  test('moveCamera calls through scrollBy', () async {
```

Comment:
I'm confused by the naming changes here; the API being called is `moveCamera` at both layers; nothing called `scrollBy` is being called in either method.

You could say "calls through with expected parameters" if you want to be more explicit about what's being verified.